### PR TITLE
Change to A record & Introduce ForwardEmail

### PR DESCRIPTION
**Pull request by sometgirldotonline**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/novafox.json [hunk 1]
```diff
@@ -4,6 +4,8 @@
         "email": "foxsdenyt@gmail.com"
     },
     "record": {
-        "CNAME": "foxsdenyt-github-io.onrender.com"
+        "A": "216.24.57.1",
```
Comment (Stef-00012) [inline domains/novafox.json:7]: record should be an array of IPs

Comment (Stef-00012) [reply domains/novafox.json:7]: Also i cannot reach your website
error 1003 Direct IP access not allowed

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/novafox.json (hunk 1 above)
Comment (novathefox) [reply domains/novafox.json:7]: oh... do u know if i can use alias records? Render's docs say alias can be used to coexist with other records

Comment (novathefox) [reply domains/novafox.json]: It's set as Alias now, :crossed_fingers: hopefully this'll work

Comment (novathefox) [reply domains/novafox.json]: the checker is failing over an older commit smh

**Comment by novathefox** [comment]
Going to remake PR

**Pull request closed without merging**